### PR TITLE
fix MainActivityTest

### PR DESCRIPTION
**Pull request by hannesa2**
We don't run into issues, because on PR we only tested lib

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -75,15 +75,17 @@ cache:
 after_failure:
   #- cat /home/travis/build/hannesa2/DbPreferences/DBPrefs/build/reports/androidTests/connected/index.html
   - export MY_MOD="DBPrefs"
-  - export MY_LOG_DIR="$(pwd)/${MY_MOD}/DBPrefs/build/reports/androidTests/connected/"
-  - export MY_LOG_DIR_RESULT="$(pwd)/${MY_MOD}/DBPrefs/build/reports/androidTest-result/connected/"
+  - export MY_LOG_DIR="$(pwd)/${MY_MOD}/build/reports/androidTests/connected/"
```
Comment (originx) [inline .travis.yml]: what was the issue, I just see that you print out some stuff and set some environment variables?

File: .travis.yml [hunk 2]
```diff
@@ -61,7 +61,7 @@ script:
   - set -o pipefail
   - ./gradlew clean build :DBPrefs:javadocJar :DBPrefs-lint:test :DBPrefs:assembleDebugAndroidTest :DBPrefs:check --profile --continue 2>&1 | tee build.log
   - set +o pipefail
-  - ./gradlew :DBPrefs:connectedAndroidTest
+  - ./gradlew connectedAndroidTest
```
Comment (hannesa2) [inline .travis.yml]: issue #1 it was never tested

File: sample/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (hannesa2) [inline sample/src/main/res/layout/activity_main.xml]: issue 2: it was not able to start

File: sample/proguard-test-rules.pro [hunk 4]
```diff
@@ -17,4 +17,7 @@
 #}
 # proguard-test.pro:
 #for easier debug if proguard fails
--keepattributes SourceFile,LineNumberTable
\ No newline at end of file
+-keepattributes SourceFile,LineNumberTable
+
+# removes Warning: library class android.graphics.drawable.Drawable depends on program class org.xmlpull.v1.XmlPullParser
+-dontwarn org.xmlpull.v1.**
```
Comment (hannesa2) [inline sample/proguard-test-rules.pro]: issue 3: there were proguard issues. But I''m not sure if just ignore it is the right way. `keepclass` was not working

File: .travis.yml [hunk 5]
```diff
@@ -75,14 +75,17 @@ cache:
...
+  - export MY_LOG_DIR="$(pwd)/${MY_MOD}/build/reports/androidTests/connected/"
+  - export MY_LOG_DIR_RESULT="$(pwd)/${MY_MOD}/build/reports/androidTest-result/connected/"
   #- sudo apt-get install -qq lynx && lynx --dump index.html > myIndex.log
   #- lynx --dump com.android.builder.testing.ConnectedDevice.html > myConnectedDevice.log
   #- lynx --dump com.android.builder.testing.html > myTesting.log
-  - pwd && cd "${MY_LOG_DIR:-.}" && pwd && ls -al
-  - for file in *.html; do echo "$file"; echo "====================="; cat "$file"; done || true
-  - for file in *.xml; do echo "$file"; echo "====================="; cat "$file"; done || true
-  - pwd && cd "${MY_LOG_DIR_RESULT:-.}" && pwd && ls -al
+  - find $MY_LOG_DIR -name "*.html" | while read -r file ; do echo $file; echo "-------"; cat $file; done
```
Comment (hannesa2) [inline .travis.yml]: her i just played around to show test output. But it's not really working well. Local it works

File: .travis.yml (hunk 2 above)
Comment (originx) [reply .travis.yml]: ah interesting, thanks for fixing

**Pull request merged**